system/lib/pthread/library_pthread.c

```diff
@@ -491,7 +488,7 @@ void* emscripten_sync_run_in_main_thread_2(
 }
 
 void* emscripten_sync_run_in_main_thread_xprintf_varargs(
-  int function, int param0, const char* format, ...) {
+  int function, long param0, const char* format, ...) {
```

Comment:
Hmm..I have not idea what this function is.. and it doesn't seem to be used anywhere.

Reply:
Removing this in #15235